integration-tests/gradle/projects/it-wasm-0/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+import org.jetbrains.dokka.gradle.DokkaTask
+import java.net.URL
+
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.dokka")
+}
+
+apply(from = "../template.root.gradle.kts")
+
+repositories {
+    maven("https://maven.pkg.jetbrains.space/kotlin/p/wasm/experimental")
```

Comment:
Can you please add a comment here, explaining that it's for the `kotlinx-coroutines-core:1.6.4-wasm1` and `atomicfu-wasm:0.18.5-wasm1` dependencies which are not publicly available yet? 

I'd like to not forget (or not be scared to) remove it once there's a public version of coroutines with wasm support